frontend/src/app/store.ts

```diff
@@ -1,10 +1,12 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
+import loginReducer from '../components/LogIn/LogInSlice';
 import counterReducer from '../components/counter/counterSlice';
 import reusableTableReducer from '../components/ReusableTable/ReusableTableSlice';
 
 export const store = configureStore({
   reducer: {
     counter: counterReducer,
+    login: loginReducer,
```

Comment:
On yesterday meeting we've decided to store token, user id and role in local storage.
... so login reducer looks be unnecessary.

Reply:
👍